nepal_compliance/nepal_compliance/report/purchase_register_ird/purchase_register_ird.py

```
@@ -94,40 +97,29 @@ def get_data(filters):
         pan = inv.invoice_pan or inv.supplier_tax_id
 
         tax_exempt = taxable_domestic_nc = taxable_import_nc = capital_taxable_amount = 0.0
+        tax_domestic_nc = tax_import_nc = tax_capital = 0.0
 
         items = items_by_invoice.get(inv.invoice, [])
+        item_vat_map = vat_breakup.get(inv.invoice, {}).get("item_vat", {})
+        row_vat = distribute_item_vat(items, item_vat_map)
 
-        for item in items:
+        for item, item_vat in zip(items, row_vat):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)